File: assignment_1/assi_1_4.cpp
#include<iostream>
using namespace std;
int main()
{
    int x;
    cin>>x;
    if(x%5==0 && x%11==0)
    {
        cout<<"YES"<<endl;
    }
    else
    {
        cout<<"NO"<<endl;
    }
}

File: assignment_1/assi_1_4.java
import java.util.*;
public class assi_1_4 
{
    public static Scanner scn = new Scanner (System.in);
    public static void main(String[] args)
    {
        int x=scn.nextInt();
        if(x%5==0 && x%11==0)
        {
            System.out.println("YES");
        }
        else
        {
            System.out.println("NO");
        }
    }
}